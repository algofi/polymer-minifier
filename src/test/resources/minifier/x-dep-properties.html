
<dom-module id="x-dep-properties">
	<template>
		<style>
		</style>
		<x-five-properties session-id="[[sessionId]]" user-id="[[userId]]" posts="{{posts}}"></x-five-properties>
		<script>
			Polymer( {
				is:'x-dep-properties',
				properties: {
					sessionId: Number,
					userId: String,
					posts: Array,
				}
			} );
			
		</script>
	</template>
</dom-module>