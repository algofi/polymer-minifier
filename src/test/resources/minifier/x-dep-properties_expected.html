<dom-module id="x-dep-properties"><template><style></style><x-a a="[[a]]" b="[[b]]" e="{{c}}"></x-a><script>
			Polymer( {
				is:'x-dep-properties',
				properties: {
					a: Number,
					b: String,
					c: Array,
				}
			} );
			
		</script></template></dom-module>